<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetOhlcSourceDlg</class>
 <widget class="QDialog" name="TargetOhlcSourceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TargetOhlcSourceDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Open:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="name_o_"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>High:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="name_h_"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Low:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="name_l_"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Close:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="name_c_"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TargetOhlcSourceDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TargetOhlcSourceDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
